src/registrar/assets/sass/_theme/_base.scss

```diff
@@ -125,3 +125,7 @@ abbr[title] {
 .flex-end {
   align-items: flex-end;
 }
+
+.remove-underline {
```

Comment:
Missing ending blank line. Also there is a USWDS .text-no-underline class (https://designsystem.digital.gov/utilities/text-styles/). You should use it. If it doesn't work because of a cascade conflict, override with eg a.text-no-underline